### PR TITLE
Add additional configuration parameters

### LINKED ISSUE #51: Document Manager configuration parameters
Document the missing configuration file options. See: https://github.com/ConsenSys/sumo/blob/main/manager/config/manager-op1-i1-docker.toml

Missing options include:

- All [vertx_config] options
- Missing [api] options
- All [api.auth] options
- All [http_client] options
- Missing [engine] options

Missing HA and Kafka options to be covered in a separate PR.

### PR DESCRIPTION
**Pull request by rolandtyler**
## Pull Request Description
Adds additional Manager configuration parameters

## Fixed Issue(s)

Fixes #51 

## Preview
https://pegasys-sumo--55.com.readthedocs.build/en/55/Reference/Configuration-File/

https://pegasys-sumo--55.com.readthedocs.build/en/55/HowTo/Configure/Configuration-File/#manager-configuration-file

### CODE REVIEW

File: docs/Reference/Configuration-File.md [hunk 1]
```diff
@@ -84,6 +156,16 @@ Details of the operator's engine component.
...
 
+#### `type`
+
+Type options are `Real` and `Fake`. The default is `Real`.
+
+!!! warning
+
+    The `Fake` option is used for testing to start the operator without the engine dependency.
+
+    This option must not be used in production.
```
Comment (jpnovais) [inline docs/Reference/Configuration-File.md:167]: Maybe worthy mentioning that it will be deprecated soon in future versions.

**Review by jpnovais — APPROVED** [review]

File: docs/Reference/Configuration-File.md [hunk 2]
```diff
@@ -64,6 +64,38 @@ ID of the Kafka consumer group for the invalid operations topic.
...
+
+This section contains the [Vert.x](https://vertx.io/docs/) settings.
+
+#### `max_event_loop_execution_time_millis`
+
+Value of max [event loop execute time](https://vertx.io/docs/apidocs/io/vertx/core/VertxOptions.html#setMaxEventLoopExecuteTime-long-), in milliseconds
+
+#### `max_worker_thread_execution_time_millis`
+
+Value of [max worker execute time](https://vertx.io/docs/apidocs/io/vertx/core/VertxOptions.html#setMaxWorkerExecuteTime-long-), in milliseconds
```
Comment (alexandratran) [inline docs/Reference/Configuration-File.md:77]:
```suggestion
Value of max [event loop execute time](https://vertx.io/docs/apidocs/io/vertx/core/VertxOptions.html#setMaxEventLoopExecuteTime-long-), in milliseconds.

#### `max_worker_thread_execution_time_millis`

Value of [max worker execute time](https://vertx.io/docs/apidocs/io/vertx/core/VertxOptions.html#setMaxWorkerExecuteTime-long-), in milliseconds.
```

File: docs/Reference/Configuration-File.md [hunk 3]
```diff
@@ -64,6 +64,38 @@ ID of the Kafka consumer group for the invalid operations topic.
...
+
+#### `log_stacktrace_threshold_millis`
+
+Threshold time in milliseconds to trigger a warning containing a [stack trace](https://vertx.io/docs/apidocs/io/vertx/core/VertxOptions.html#setWarningExceptionTime-long-).
+
+#### `http_verticles`
+
+Number of processing units, for HTTP requests.
```
Comment (alexandratran) [inline docs/Reference/Configuration-File.md:85]:
```suggestion
Number of processing units for HTTP requests.
```

(Make consistent with the following section.)

File: docs/Reference/Configuration-File.md [hunk 4]
```diff
@@ -64,6 +64,38 @@ ID of the Kafka consumer group for the invalid operations topic.
...
+
+!!! important
+
+    Do not set higher than the max number of CPUs in the underlying machine.
```
Comment (alexandratran) [inline docs/Reference/Configuration-File.md:89]:
```suggestion
    Don't set `http_verticles` higher than the maximum number of CPUs in the underlying machine.
```

File: docs/Reference/Configuration-File.md [hunk 5]
```diff
@@ -64,6 +64,38 @@ ID of the Kafka consumer group for the invalid operations topic.
...
+
+#### `websocket_verticles`
+
+Number of processing units for Websockets requests.
```
Comment (alexandratran) [inline docs/Reference/Configuration-File.md:93]:
```suggestion
Number of processing units for WebSocket requests.
```

File: docs/Reference/Configuration-File.md [hunk 6]
```diff
@@ -64,6 +64,38 @@ ID of the Kafka consumer group for the invalid operations topic.
...
+
+!!! important
+
+    Do not set higher than the max number of CPUs in the underlying machine.
```
Comment (alexandratran) [inline docs/Reference/Configuration-File.md:97]:
```suggestion
    Don't set `websocket_verticles` higher than the maximum number of CPUs in the underlying machine.
```

File: docs/Reference/Configuration-File.md [hunk 7]
```diff
@@ -76,6 +108,46 @@ HTTP JSON-RPC listening port to access the manager's APIs.
...
+
+JSON-RPC API version.
+
+### `api.auth`
+
+Details on API authorizations.
+
+#### `rollup_account_max_token_duration`
+
+Max time a Rollup account session can last. Tokens are issued on login and cannot have a duration time longer than this.
```
Comment (alexandratran) [inline docs/Reference/Configuration-File.md:125]:
```suggestion
Maximum time a Rollup account session can last. Tokens are issued on login and can't have a duration time longer than this.
```

File: docs/Reference/Configuration-File.md [hunk 8]
```diff
@@ -76,6 +108,46 @@ HTTP JSON-RPC listening port to access the manager's APIs.
...
+
+#### `api_account_token_duration`
+
+Duration of API accounts tokens.
+
+!!! note
+
+    API accounts are special accounts that can read the state of all Rollup Accounts. Credentials & Permissions for these accounts are specified in a credentials.toml file.
```
Comment (alexandratran) [inline docs/Reference/Configuration-File.md:133]:
```suggestion
    API accounts are special accounts that can read the state of all Rollups accounts. Credentials and permissions for these accounts are specified in a `credentials.toml` file.
```

File: docs/Reference/Configuration-File.md [hunk 9]
```diff
@@ -76,6 +108,46 @@ HTTP JSON-RPC listening port to access the manager's APIs.
...
+
+#### `clock_leniency`
+
+Amount time desynchronization tolerated between the users and server for JWT token validation.
```
Comment (alexandratran) [inline docs/Reference/Configuration-File.md:137]:
```suggestion
Amount of time desynchronization tolerated between the users and server for JWT token validation.
```

File: docs/global/Manager-Configuration-File.md [hunk 10]
```diff
@@ -2,31 +2,51 @@
...
     [ha.kafka]
     brokers = "kafka-broker:29092"
-    pending_operations_replication_topic = "operator1_i0_pending_operations"
+    pending_operations_replication_topic = "operator1_i1_pending_operations"
     pending_operations_replication_partitions = 3
-    consumer_group_name = "operator1_i0_pending_op_consumer"
-    pending_operations_replication_topics_to_copy = ["operator1_i1_pending_operations"]
+    consumer_group_name = "operator1_i1_replicatior"
+    pending_operations_replication_topics_to_copy = ["operator1_i0_pending_operations"]
     ```
```
Comment (alexandratran) [inline docs/global/Manager-Configuration-File.md:52]: For consistency, can you also remove the four spaces at the beginning of the lines in the sample Engine configuration files on the pages `HowTo/Configure/Configuration-File` and `Reference/Configuration-File/#engine-configuration`?

**Review by alexandratran — APPROVED** [review]

**Pull request merged**